### ISSUE TITLE
Speakers unresponsive after power failure

### ISSUE THREAD

**Issue by jaytavares** [jaytavares/cinesense#1]
Following loss of power, speakers will no longer power on.

To reproduce, unplug speaker power cable with Cinesense installed and working. Upon restoring power, LED on interface module will flash for 10 seconds as if muted. After that, the speakers will remain unresponsive to audio and IR remote control signals.

I'm still investigating the root cause of this issue. With the Cinesense installed, a power failure seems to trigger some kind of lock out mode in the speakers. While locked out, the speakers no longer respond to IR commands. Since this is how the Cinesense controls the speakers, this means that the speakers cannot be powered on automatically or via the remote control.

**To resolve the issue:**
1. Unplug the speaker's power cord from the wall.
1. Remove the Cinesense adapter from your speakers.
1. Reattach interface module cable directly to subwoofer, leaving Cinesense disconnected. 
1. Plug power cord back into the wall.
1. Wait for the LED on the interface module to stop flashing.
1. Reinstall the Cinesense adapter while keeping power cord plugged in.

**Comment by jaytavares** [comment]
![Power failure error condition](https://user-images.githubusercontent.com/711127/82979926-3f1dc000-9fb6-11ea-8246-8a3348a35bba.png)

**Comment by jaytavares** [comment]
I figured out the cause of this issue today and fixed it. The culprit is the pull up resistor on the CONTROL_IN line. In my troubleshooting, I discovered that the presence of positive voltage on that line during power up would result in the behavior described above. The behavior was not exhibited when the line was held low or left floating. To fix the issue, I added an initialization step to the setup routing that holds the line low for four seconds. That seems to have fixed it.
